doc/CONTRIBUTING.md

```diff
@@ -106,6 +108,13 @@ Don't worry about setting DynamoDB endpoints or creating tables: the broker's co
 
 **Note:** The local version of DynamoDB is not recommended for production.
 
+### Install RabbitMQ
+
+RabbitMQ is used to pass jobs to the validator, and requires Erlang to be installed before RabbitMQ.
```

Comment:
It does work on Windows 10, and for Windows 8 we have one failure and one success.  The statement from Pivotal was "We've seen this reported on Windows a few times but don't know how to reproduce."  So we don't know how widespread it is and we don't know that it is specific to Windows 8.  Broader testing would be helpful, not sure what the process for that would be.